src/api/app/views/webui2/webui/interconnects/new.html.haml

```diff
@@ -1,6 +1,54 @@
 :ruby
   @pagetitle = 'Interconnect'
+  opensuse_exists = Project.exists?(name: 'openSUSE.org')
+  tizen_exists = Project.exists?(name: 'tizen.org')
+  packman_exists = Project.exists?(name: 'Packman')
 
 .card.mb-3
   = render partial: 'tabs'
   .card-body
+    %h3 Connect a remote Open Build Service instance
+    %span Use a public OBS for default base distributions
+    .my-3
+      = button_tag('openSUSE', class: "interconnect mr-1 mb-1 btn #{opensuse_exists ? 'btn-outline-dark' : 'btn-outline-primary'}",
```

Comment:
The ternary operator here could be replaced by some CSS like this:

```css
button.interconnect {
  &:not([disabled="disabled"]) {
    @extend .btn-outline-primary;
  }

  &[disabled="disabled"] {
    @extend .btn-outline-dark;
  }
}
```

Reply:
Nice!, thanks ^^